libkbfs/journal_block_server.go

```diff
@@ -130,6 +130,6 @@ func (j journalBlockServer) IsUnflushed(ctx context.Context, tlfID tlf.ID,
 }
 
 func (j journalBlockServer) Shutdown() {
-	j.jServer.shutdown()
+	j.jServer.shutdown(context.Background())
```

Comment:
Should we add `ctx` to `BlockServer.Shutdown()` for this purpose?  Would be nice, but I don't care too much.